dynamic_models/models.py

```diff
@@ -13,13 +13,19 @@
 class ModelSchema(models.Model):
     name = models.CharField(max_length=32, unique=True)
     db_name = models.CharField(max_length=32, default=DEFAULT_DB_ALIAS)
+    is_managed = models.BooleanField(default=True)
+    use_applable_as_table_prefix = models.BooleanField(default=True)
```

Comment:
**suggestion**: I think we should replace these with the same fields as on `Meta` so it's very obvious what they do.

```suggestion
    managed = models.BooleanField(default=True)
    db_table = models.CharField(null=True, max_length=64)
```

Exposing `db_table` instead is also more flexible and let's the user use any naming convention they want instead of only allowing a toggle on the app label prefix.

If `db_table` is null we should fall back to the current default, which should be the same as Django's default.